3D/senior-csharp-test/Assets/Cache/FileCacheStorage.cs

```diff
@@ -52,16 +56,17 @@ private void InitCache()
 			return;
 		}
 
-		if( TryLoadCacheEntries() )
+		if( DeserializeCacheEntries() )
 		{
-			CheckCacheVersion();
-			ClearExpiredEntries();
+			if( !IsCurrentVersion() )
+				DeleteAll();
+			ClearExpiredCacheEntries();
```

Comment:
shouldn't this be in a `else`? 🤔

Reply:
Indeed (but that's like that on Display)

Reply:
done